crates/bevy_ecs/src/system/function_system.rs

```diff
@@ -421,6 +426,19 @@ where
             self.system_meta.name.as_ref(),
         );
     }
+    fn default_labels(&self) -> Vec<Box<dyn SystemLabel>> {
+        vec![self.func.as_system_label()]
+    }
+}
+
+#[derive(Debug, Hash, PartialEq, Eq, Copy, Clone)]
+/// A [`SystemLabel`] that was automatically generated for a system on the basis of its `TypeId`.
+pub struct SystemTypeIdLabel(pub(crate) TypeId);
```

Comment:
```suggestion
pub(crate) struct SystemTypeIdLabel(pub(crate) TypeId);
```
Or if we can afford it, remove the `pub` entirely.

Like #4250. This is more of an implementation detail than anything else, probably shouldn't made public.

Reply:
Imo this label is a part of our public api (especially with my recent changes, which remove the box from AsSystemLabel). I think there might be cases where this should be accessible to users.